pkg/db/sqlite.go

```diff
@@ -118,3 +118,18 @@ func (m *sqlite) Initialize() error {
 // This is not yet supported for sqlite
 func (m *sqlite) PrintEnvironmentTestsAndTestCases(_ http.ResponseWriter, _ *http.Request) {
 }
+
+// ServeEnvCharts writes the overall environment charts to a JSON HTTP response
+// This is not yet supported for sqlite
+func (m *sqlite) ServeEnvCharts(_ http.ResponseWriter, _ *http.Request) {
+}
```

Comment:
This is going to return a 200 even though nothing happened, should return a 501 here. Same with the two below

Reply:
Made the handler return a StatusNotImplemented when there is no error getting data but the data is nil as a solution as the get functions for the data for each chart are encapsulated from the http